source/ddbus/thin.d

```diff
@@ -27,6 +26,19 @@ struct ObjectPath {
     return _value;
   }
 
+  /// Returns the string representation of this ObjectPath
+  string value() const pure nothrow @safe {
+    return _value;
+  }
+
+  /// Sets and validates the string representation of this ObjectPath
+  string value(string newVal) @safe {
```

Comment:
As `ObjectPath` is essentially just a wrapped string reference, there is no allocation in `objPath = ObjectPath("/some/path")`, so no advantage in setting the value.

Also, I like to have the ObjectPath construction be explicit, because it is not nothrow.

Reply:
hm you have a point there, sounds like the better alternative